### DISCUSSION TITLE
Timeline for next release?

### DISCUSSION THREAD

**Discussion by alexkirsz** [Ogeon/palette#203] [General]
Hello!

I'd love to use the new ColorDifference trait and the Delta-E/CIEDE2000 algorithm. It was merged last year in https://github.com/Ogeon/palette/pull/162 but there hasn't been a release since then. Is this waiting on something else?

Cheers,
Alex

**Comment by Ogeon** [comment]
Hi!

It's primarily waiting for me to finish #192, but I haven't been feeling like programming much on my free time for the past few weeks. This happens when I have a lot going on at my day job, but it's starting to calm down for me, so I should be back at this soon. Other than that, I have decided to rename the `Limited` trait to `Clamp`. But that's about it. So, the blocker is just me being slow at the moment. 🙂

**Comment by Ogeon** [comment]
Hello again! Short update. I have organized the plan in a milestone you can keep an eye on: https://github.com/Ogeon/palette/milestone/5. I don't intend for it to scope creep any further, unless something important or severe enough shows up. Picking up issues from there is also the best way to help pushing it forwards, in case anyone is wondering!

**Comment by Ogeon** [comment]
Release progress update: It's done! https://ogeon.github.io/2021/07/12/palette-0.6.0.html
